pkg/sse/stream.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sse
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"math"
+	"net/http"
+	"strings"
+	"time"
+
+	oktetoLog "github.com/okteto/okteto/pkg/log"
+)
+
+var (
+	// gitDeployUrlTemplate (baseURL, namespace, dev environment name, action name)
+	GitDeployUrlTemplate = "%s/sse/logs/%s/gitdeploy/%s?action=%s"
```

Comment:
This would make sense to be in `okteto/sse.go`. This file/package is agnostic on the endpoint